marklogic-data-hub-central/ui/e2e/cypress/integration/modeling/writerScenario1.spec.tsx

```diff
@@ -173,6 +173,28 @@ describe("Entity Modeling Senario 1: Writer Role", () => {
     propertyTable.getSortIcon("nicknames").should("exist");
     modelPage.getEntityModifiedAlert().should("exist");
   });
+  it("Verify that the tooltips exists in property table", () => {
```

Comment:
Can we actually handle these with RTL in property-table.test.tsx? To take some of the load off of cypress

Reply:
@briantang Done